5_4_A.py

```diff
@@ -0,0 +1,35 @@
+def interleave(*iterables):
+    """
+    Standard version
+    :param iterables: Unknown amount of iterables
+    :return: the list of separated members of the iterables

```

Comment:
1. Capital letter :)
2. Can you rephrase this please? its a bit not understandable.